deploy/base/webhook/certificate.yaml

```diff
@@ -13,8 +13,8 @@ metadata:
   namespace: security-profiles-operator
 spec:
   dnsNames:
-  - profilebinding-webhook-service.security-profiles-operator.svc
-  - profilebinding-webhook-service.security-profiles-operator.svc.cluster.local
+  - profile-webhook-service.security-profiles-operator.svc
+  - profile-webhook-service.security-profiles-operator.svc.cluster.local
```

Comment:
nit: what about just "webhook-service"? We'll probably add more webhook capabilities in the future. (At least I'm already working on one extra addition).

Reply:
Fixed as suggested :+1: